src/main/java/taxi/controller/page/CreateDriverController.java

```diff
@@ -0,0 +1,53 @@
+package taxi.controller.page;
```

Comment:
why `page`? if you want to split your controllers, do it by the car/driver/authenthication packages